great_tables/_gt_data.py

```diff
@@ -81,6 +82,16 @@ def _replace(self, **kwargs: Any) -> Self:
 
         return new_obj
 
+    def _add_google_font_import(self, import_statement: str) -> Self:
+        if import_statement not in self._google_font_imports:
+            new_imports = self._google_font_imports.copy()
+            new_imports.add(import_statement)
+            return self._replace(_google_font_imports=new_imports)
+        return self
+
+    def _get_google_font_imports(self) -> list[str]:
```

Comment:
This is only used once, so I would get rid of the function and use the code directly

Reply:
Now removed.